src/main/java/net/mcreator/element/types/Dimension.java

```diff
@@ -73,6 +73,7 @@
 	public Sound portalSound;
 	public boolean enableIgniter;
 	public String igniterName;
+	public String rarity;

```

Comment:
This parameter does not address dimension rarity, but igniter rarity, so name accordingly (igniterRarity)

Reply:
Ok

Reply:
Done